android/KMAPro/kMAPro/src/main/AndroidManifest.xml

```diff
@@ -27,6 +27,9 @@
     <!--    how to enable Sentry's debug mode-->
     <meta-data
       android:name="io.sentry.debug" android:value="true" />
+    <!-- Disable monitoring "Application Not Responding" (ANR).  Too many reports at the default 4000 ms -->
```

Comment:
Hmm. 4000ms is a very long delay on usage. Can we turn this into an issue?

Reply:
k, this will be #2811